source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -44,6 +44,49 @@
 from enum import Enum
 import itertools
 
+# Possible values implemented. Full languages within the doc:
+# https://developer.here.com/documentation/routing/dev_guide/topics/resource-param-type-languages.html#languages
+# Be careful, the syntax has to be exact
+class Languages(Enum):
+    afrikaans = "af"
+    arabic = "ar-sa"
+    chinese = "zh-cn"
+    dutch = "nl-nl"
+    english = "en-gb"
+    french = "fr-fr"
+    german = "de-de"
+    hebrew = "he"
+    hindi = "hi"
+    italian = "it-it"
+    japanese = "ja-jp"
+    portuguese = "pt-pt"
+    russian = "ru-ru"
+    spanish = "es-es"
+
+
+def _convert_here_language(language):
```

Comment:
something interesting about enum is that, the `[]` is implemented. you can do `Languages['af']`, so the convert function is not necessary, and if the keyword doesn't existe, it will raise an keyerror exception, just like a `map`

Reply:
Thanks for the tip. Very usefull :heart: